src/workerd/io/compatibility-date.c++

```diff
@@ -91,8 +119,12 @@ struct CompatDate {
 
   static CompatDate today() {
     time_t now = time(nullptr);
+#ifdef _MSC_VER
+    auto t = *gmtime(&now);
```

Comment:
This isn't thread-safe. Do we care? I guess we're single-threaded right now (in workerd, at least), but not sure if that's a good assumption forever.